internal/pkg/skuba/deployments/ssh/cri.go

```diff
@@ -114,3 +117,47 @@ func criStart(t *Target, data interface{}) error {
 	_, _, err := t.ssh("systemctl", "enable", "--now", "crio")
 	return err
 }
+
+// CriMigrate migrates the old configuration of cri-o < 1.17 to the new format.
+func CriMigrate() error {
+	if err := criGenerateLocalConfiguration(); err != nil {
+		return err
+	}
+
+	if err := criRemoveLocalOldFile(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func criGenerateLocalConfiguration() error {
+	cfg := clusterinit.InitConfiguration{
+		PauseImage:        kubernetes.ComponentContainerImageForClusterVersion(kubernetes.Pause, kubernetes.LatestVersion()),
```

Comment:
is LatestVersion correct, or should that be the _next_ version?

Reply:
At least this is giving me this result: `registry.suse.de/devel/caasp/5/containers/containers/caasp/v5/pause:3.2`, which I believe is fine, no?

Reply:
It is fine for now, but it might not be in the future, if the user is lagging behind more than a version. No?

Reply:
Well, in this case they should be using the proper `skuba` for each scenario (same as other upgrades). I don't think we have a `next` version somewhere in the code, right?

Reply:
Mmm interesting, I thought we had that, because we have a path in the `plan` and `apply` commands, when dealing with multiple k8s versions, but I didn't check the code.